scripts/aks-auto-start.sh

```diff
@@ -29,11 +29,36 @@ jq -c '.[]' <<< $SUBSCRIPTIONS | while read subscription; do
         statuscode=$(curl -s -o /dev/null -w "%{http_code}"  https://$APP.$ENVIRONMENT.platform.hmcts.net)
 
         if [[ $statuscode -eq 200 ]]; then
-            echo "$APP works in $ENVIRONMENT after $NAME start-up"
+            if [[ "$ENVIRONMENT" == "Sandbox" || "$ENVIRONMENT" == "sandbox" ]]; then
```

Comment:
Please simplify the code so that you create a mapping function for environments and then just do a curl that has aks-monitor-$environment as the channel